src/components/CouponsPageHeader/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
I would argue that there's no need for this component since it's only 1 line of HTML